core/src/language/go.rs

```diff
@@ -413,7 +413,7 @@ impl Go {
                         "\t{} {} = {:?}",
                         variant_type_const,
                         variant_key_type,
-                        &v.shared().id.original
+                        &v.shared().id.renamed
```

Comment:
What happens in the default case, where no serde renaming occurs? What casing is used then?

Reply:
If no serde renaming occurs, it will match the original one, which is TItleCase.